### PR TITLE
Consolidate relying party revocation requirements

### LINKED ISSUE #90: Harmonize requirements for Relying Parties to check revocation
We specify the same, or similar, requirements on Relying Parties in several places:

CPS
> [1.3.4 Relying parties](https://letsencrypt.org/documents/isrg-cps-v3.3/#1.3.4-relying-parties)
> Relying Parties must verify the validity of certificates via CRL or OCSP prior
> to relying on certificates. CRL and OCSP location information is provided
> within certificates.

> [4.5.2 Relying party public key and certificate usage](https://letsencrypt.org/documents/isrg-cps-v3.3/#4.5.2-relying-party-public-key-and-certificate-usage)
> Relying Parties ignoring certificate expiration, revocation data provided via
> OCSP or CRL, or other pertinent information do so at their own risk.

> [4.9.6 Revocation checking requirement for relying parties](https://letsencrypt.org/documents/isrg-cps-v3.3/#4.9.6-revocation-checking-requirement-for-relying-parties)
> Relying Parties who cannot or choose not to check revocation status, but
> decide to rely on a certificate anyway, do so at their own risk.

We should consolidate on one of these, probably 4.9.6. The others can have "See section 4.9.6" (or simply have the sentence deleted).

### PR DESCRIPTION
**Pull request by aarongable**
Fixes #90

### CODE REVIEW

File: CPS.md [hunk 1]
```diff
@@ -559,7 +555,7 @@ Investigation into a revocation request will begin within 24 hours of receiving
 
 ### 4.9.6 Revocation checking requirement for relying parties
 
-Relying Parties who cannot or choose not to check revocation status, but decide to rely on a certificate anyway, do so at their own risk.
+Relying parties must verify the validity of certificates via CRL or OCSP prior to relying on certificates. Relying Parties who cannot or choose not to check certificate expiration or revocation status, but decide to rely on a certificate anyway, do so at their own risk.
```
Comment (bdaehlie) [inline CPS.md:558]: I don't think we should say "must verify" here as it sounds like a hard requirement when it isn't (so far as I know). Subscribers are allowed to ignore revocation info if they choose to, they just do it at their own risk. Maybe change "must" to "should"?

Also, we need to capitalize the second word of this paragraph as "Relying Parties" is a defined term.

Comment (aarongable) [reply CPS.md:558]: We have a similar "must" statement in 4.5.2:
> Relying Parties must fully evaluate the context in which they are relying on certificates and the information contained in them, and decide to what extent the risk of reliance is acceptable. If the risk of relying on a certificate is determined to be unacceptable, then Relying Parties should not use the certificate or should obtain additional assurances before using the certificate.

Should we update that one as well, or continue to use "must" in both?

Comment (bdaehlie) [reply CPS.md:558]: Good catch, let's change that to "should" as well. I don't see anything in the BRs or the Mozilla CA Policy that requires us to say "must" in this context.

Comment (aarongable) [reply CPS.md]: Done!

File: CPS.md [hunk 2]
```diff
@@ -559,7 +555,7 @@ Investigation into a revocation request will begin within 24 hours of receiving
 
 ### 4.9.6 Revocation checking requirement for relying parties
 
-Relying Parties who cannot or choose not to check revocation status, but decide to rely on a certificate anyway, do so at their own risk.
+Relying parties should verify the validity of certificates via CRL or OCSP prior to relying on certificates. Relying Parties who cannot or choose not to check certificate expiration or revocation status, but decide to rely on a certificate anyway, do so at their own risk.
```
Comment (bdaehlie) [inline CPS.md:558]: Still need to fix capitalization of "parties" in the second word of this paragraph.

Comment (aarongable) [reply CPS.md]: Fixed

**Pull request merged**